src/mpid/ch4/shm/src/shm_impl.c

```diff
@@ -4,11 +4,8 @@
  *      See COPYRIGHT in top-level directory.
  */
 
-#include "mpl.h"
-
-MPL_SUPPRESS_OSX_HAS_NO_SYMBOLS_WARNING;
-
 #include "posix_impl.h"
+MPL_SUPPRESS_OSX_HAS_NO_SYMBOLS_WARNING;
```

Comment:
Do we really need to move this line here? Wouldn't removing the inclusion of mpl.h be sufficient?

Reply:
It needs to happen after "#include <mpl.h>", but we need control when <mpl.h> gets included, ie ensure it is after `mpix.h`. Therefore it needs to move after "posix_impl.h" which includes "mpiimpl.h", which include mpl.h there.